### PR TITLE
Update dashlane from 6.1914.0.19481 to 6.1918.1.20296

### PR DESCRIPTION
**Pull request by core-code**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by core-code** [comment]
actually not sure about that warning, would have to debug it

File: Casks/dashlane.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'dashlane' do
-  version '6.1914.0.19481'
```
Comment (vitorgalvao) [inline Casks/dashlane.rb]: I see this version in the appcast.

Comment (core-code) [reply Casks/dashlane.rb]: thats weird, i am certainly seeing '6.1918.1.20296' in the appcast every time i check. are you using the URL from the cask?

Comment (core-code) [reply Casks/dashlane.rb]: btw the download version from the homepage is already even newer, as can be seen in this config file:

https://ws1.dashlane.com/5/binaries/query?format=json&os=OS_X_10_14_5&currentSoftwareVersion=6.1850.0.16084&partner=&target=archive&platform=launcher_macosx

Comment (vitorgalvao) [reply Casks/dashlane.rb]: > are you using the URL from the cask?

Yes.



> btw the download version from the homepage is already even newer, as can be seen in this config file

Any objection to replacing the appcast with that? The link can be further shortened: `https://ws1.dashlane.com/5/binaries/query?format=json&os=OS_X_10_14_5&target=archive&platform=launcher_macosx`.

Comment (core-code) [reply Casks/dashlane.rb]: no objections from my side, though i still find it weird we are seeing different results from the appcast. must be some form of staging.

**Pull request merged**